Dockerfile

```diff
@@ -1,6 +1,6 @@
-FROM java:8-jre
+FROM openjdk:8u131-jre-alpine
```

Comment:
Well the old image is deprecated, so you won't get any new updates on it. Thought it would be useful to put in anyway :P